scripting/get5.sp

```diff
@@ -417,6 +426,8 @@ public void OnPluginStart() {
   g_OnSeriesInit = CreateGlobalForward("Get5_OnSeriesInit", ET_Ignore);
   g_OnSeriesResult =
       CreateGlobalForward("Get5_OnSeriesResult", ET_Ignore, Param_Cell, Param_Cell, Param_Cell);
+  g_OnSeriesResultWithoutWinners =
```

Comment:
But `Get5_OnSeriesResult` expects to receive `team1MapScore` and `team1MapScore` which in this case are both 0 because there's no winner. Also, it gets the stats, which does not exists too.

Do you think that this `Get5_OnSeriesResultWithoutWinners` function is an overkill?